packages/circuit-ui/hooks/useCollapsible/useCollapsible.stories.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2021, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Body from '../../components/Body';
+import Button from '../../components/Button';
+import { spacing } from '../../styles/style-mixins';
+
+import { CollapsibleOptions, useCollapsible } from './useCollapsible';
+import docs from './useCollapsible.docs.mdx';
+
+export default {
+  title: 'Hooks/useCollapsible',
+  parameters: {
+    docs: { page: docs },
+  },
+};
+
+export const Example = (args: CollapsibleOptions) => {
+  const { isOpen, getButtonProps, getContentProps } = useCollapsible(args);
+
+  return (
+    <section>
+      <Button {...getButtonProps()}>
```

Comment:
Maybe we should add a story with a custom Button onClick/custom Body styles? What do you think?

Reply:
I kept the stories for the hooks very basic on purpose to keep the focus on the relevant code. If we add additional code, the source view becomes convoluted and makes it harder for developers to figure out which parts of the code are relevant and necessary. 

I'm planning to add a link to the SideNavigation as an example implementation. Do you think that would be enough?

Reply:
I think so! Thanks 🙂